File: FDP/T3_A2_PinedoBerumenClaraRuby/src/Ejercicio10_número_mayor.java

import java.util.Scanner;

public class Ejercicio10_número_mayor {

	public static void main(String[] args) {
	
Scanner entrada = new Scanner (System.in);
		
		double num1 = 0.0;
		double num2 = 0.0;
		double num3 = 0.0;
		double max = 0.0;
		
		System.out.println("Ingrese el valor del número 1: ");
		num1 = entrada.nextDouble();
		
		System.out.println("Ingrese el valor del número 2: ");
		num2 = entrada.nextDouble();
		
		System.out.println("Ingrese el valor del número 3: ");
		num3 = entrada.nextDouble();
		
		max = Math.max((Math.max(num1,num2)),num3);
		
		
		if(num1==max) {
			System.out.println("El mayor es el número 1  ");
		}
		
		if(num2==max) {
			System.out.println("El mayor es el número 2  ");
		}
		
		if(num3==max) {
			System.out.println("El mayor es el número 3  ");
		}
		
		

	}

}

File: FDP/T5_A3_PinedoBerumenClaraRuby/Ejercicio10_Número_mayor.py
'''
Created on 28 nov. 2019

@author: Ruby
'''
print("===Programa que lee tres numeros e indica cual es el mayor===\n")


        
num1 = float(input("Ingrese el valor del numero 1: "))
num2 = float(input("Ingrese el valor del numero 2: "))
num3 = float(input("Ingrese el valor del numero 3: "))
        
if(num1>num2 and num1>num3):
    print("El mayor es el numero 1  ")
        
elif(num2>num1 and num2>num3):
    print("El mayor es el numero 2  ")
        
elif(num3>num2 and num3>num1):
    print("El mayor es el numero 3  ")
        
